.github/workflows/runbook-freshness-check.yaml

```
@@ -0,0 +1,237 @@
+                
+                issuesCreated.push(`#${issue.data.number} - ${fileName}`);
+                console.log(`✅ Created new issue #${issue.data.number} for ${fileName}`);
+              }
+            }
+            
+            // Summary
+            console.log(`\n📊 Summary:`);
+            console.log(`- Found ${staleRunbooks.length} stale runbook${staleRunbooks.length > 1 ? 's' : ''}`);
+            console.log(`- Created ${issuesCreated.length} new issue${issuesCreated.length > 1 ? 's' : ''}`);
+            console.log(`- Updated ${issuesUpdated.length} existing issue${issuesUpdated.length > 1 ? 's' : ''}`);
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)